src/amber/cli/config.cr

```diff
@@ -0,0 +1,27 @@
+module Amber::CLI
+  module Config
+    def self.language
+      amber_yml["language"].to_s
+    rescue
+      "slang"
+    end
+
+    def self.database
+      amber_yml["database"].to_s
+    rescue
+      "pg"
+    end
+
+    def self.model
+      p amber_yml["model"]
+      amber_yml["model"].to_s
+    rescue
+      "granite"
+    end
+
+    private def amber_yml
+      raise "Missing Amber YAML file" unless File.exists?(AMBER_YML)
```

Comment:
Again, I don't think raising an exception is the way to go here. At least not if it is going to be rescued by the methods above.

```crystal
private def amber_yml
  return nil unless File.exists?(AMBER_YML)
  @@amber_yml ||= YAML.parse(File.read AMBER_YML)
end
```

Reply:
This makes me thing that the config should be a class instead of a module.

Reply:
High level, I think parsing a file is the job of a module and holding config information is the job of a class.

We have other yaml files that are loaded and written, correct? There could be a module that takes care of all of that, and a config class could leverage that module.